core/services/log/orm.go

```diff
@@ -81,17 +129,118 @@ func (o *orm) MarkBroadcastConsumed(blockHash common.Hash, blockNumber uint64, l
 		return errors.Wrap(err, "MarkBroadcastConsumed failed")
 	}
 	if rowsAffected == 0 {
-		return errors.Errorf("cannot mark log broadcast as consumed: does not exist")
+		return errors.Wrap(err, "while marking log broadcast as consumed")
+	}
+	return nil
+}
+
+func (o *orm) RemoveUnconsumedSetPending(qopts ...postgres.QOpt) (*int64, error) {
```

Comment:
i guess in general errors could be wrapped in this method?

Reply:
Yeah I think I hesitated since it's just calling helpers that add their own error messages so I don't know what else there is to say, but open to suggestions.

Reply:
it might be fine, just thought about a wider suggestion on wrapping

Reply:
Can we have a brief comment explaining in human language what this function does?

Reply:
I generally document the interface - is this clear?
```go
// RemoveUnconsumedSetPending cleans up the database by removing any unconsumed broadcasts and updating the pending
// broadcasts block number if necessary, which is returned as well.
RemoveUnconsumedSetPending(qopts ...postgres.QOpt) (lowestBlockNum *int64, err error)
```
Working on adding more inline implementation detail comments though.

Reply:
Renamed `Reinitialize()` and documented.